icon_validator/exceptions.py

```diff
@@ -0,0 +1,5 @@
+class ValidationException(Exception):
```

Comment:
It would be cool if this exception could report back which validator rule failed